roles/kubernetes/preinstall/tasks/0040-verify-settings.yml

```diff
@@ -160,13 +132,6 @@
     - dashboard_enabled
     - not ignore_assert_errors
 
-- name: Stop if kernel version is too low
```

Comment:
why?

Reply:
Same that the mem requirements, this is checked by kubeadm. I'll drop this as well per above.